recipes/r-xcell/meta.yaml

```diff
@@ -0,0 +1,62 @@
+package:
+  name: r-xcell
+  version: "1.2"
+
+source:
+  url:
+    - https://github.com/dviraran/xCell/archive/1.2.tar.gz
+  sha256: 52dcd5cfb5c8a4870cb485ee57f34d7830b23bdc61fd3012d859fdcc22ad941c
+
+build:
+  number: 0
```

Comment:
using script: here you can remove the build file

Reply:
sorry, I'm not sure what you mean...

Reply:
got it^^
Thanks @epruesse!